app-ios/Sources/AnnouncementFeature/AnnouncementItemView.swift

```diff
@@ -0,0 +1,32 @@
+import appioscombined
+import Assets
+import SwiftUI
+
+struct AnnouncementItemView: View {
+
+    let announcement: Announcement
+
+    var body: some View {
+        HStack(alignment: .top, spacing: 0) {
+            Image(announcement.type, bundle: .myModule)
+                .resizable()
+                .frame(width: 16, height: 16)
+            Spacer().frame(width: 8)
+            VStack(alignment: .leading) {
+                Text(announcement.title)
+                    .font(.system(size: 16, weight: .semibold))
+                Spacer().frame(height: 8)
```

Comment:
How about using "spacing:" instead of "Spacer" ?

https://developer.apple.com/documentation/swiftui/vstack

```suggestion
            VStack(alignment: .leading, spacing: 8) {
                Text(announcement.title)
                    .font(.system(size: 16, weight: .semibold))
```